modules/monix/test/src/weaver/monixcompat/MonixSuiteTest.scala

```diff
@@ -0,0 +1,63 @@
+package weaver.monixcompat
+
+import cats.effect.Resource
+
+import weaver.framework.DogFood
+
+import monix.eval.Task
+import sbt.testing.Status
+
+object MonixSuiteTest extends MonixSuite {
+  type Res = Unit
+  override def sharedResource: Resource[Task, Unit] = Resource.pure(())
```

Comment:
TODO: Test resource access patterns as in ZIO combat tests.

Reply:
Not necessary 😄 , you're extending `MutableFSuite` which is already tested. zio has specific tests because it extends an interface that's higher in the hierarchy, which does not make use of `cats.effect.Resource`

Reply:
I found it a bit hard to follow the base framework tests, so I wasn't sure. 
In that case, I can replace `MonixSuite` here with `SimpleMonixSuite` here. Since that is essentially what it is.